jpa/spec-tests/src/main/java/ee/jakarta/tck/persistence/core/persistenceUnitUtil/Client.java

```diff
@@ -30,14 +32,16 @@ public class Client extends PMClientBase {
 
 	private static final Logger logger = (Logger) System.getLogger(Client.class.getName());
 
+	private final Employee empRef[] = new Employee[2];
+
 	public Client() {
 	}
 
 	public JavaArchive createDeployment() throws Exception {
 
 		String pkgNameWithoutSuffix = Client.class.getPackageName();
 		String pkgName = pkgNameWithoutSuffix + ".";
-		String[] classes = { pkgName + "Employee" };
+		String[] classes = { pkgName + "Employee", "ee.jakarta.tck.persistence.core.versioning.Member" };
```

Comment:
`Member.class.getName()` would look better

Reply:
`Member.class.getName()` yes, it will better handle some refactoring in the future.